### ISSUE TITLE
Error: React is not defined

### ISSUE THREAD

**Issue by btables** [ChristopherBiscardi/gatsby-mdx#224]
**Describe the bug**

I am trying to get a simple example of gatsby-mdx working in my repo. Whenever I try and use <MDXRenderer>, I am getting an error saying React is not defined (node_modules/gatsby-mdx/mdx-renderer.js:15)

This is my first time using gatsby-mdx, I installed by running

  npm install gatsby-mdx @mdx-js/mdx @mdx-js/tag

I'm not really sure where to start looking for the issue... but if it means anything, I did play around a little and try to lower the version (0.2.0 -> 0.3.1-ci.225) by running

  npm install gatsby-mdx@0.3.1-ci.225

And my page loaded fine!

**To Reproduce**
https://github.com/btables/gatsby-mdx-react-undefined

All changes to cause the error are in the second commit (the first is just to set up a gatsby starter site). 
1. run 'npm install'
2. run 'gatsby develop'
3. navigate  to 'http://localhost:8000/blogs/blog1/'
5. Stop gatsby develop
6. run 'npm install gatsby-mdx@0.3.1-ci.225'
7. run 'gatsby develop'
8. navigate  to 'http://localhost:8000/blogs/blog1/'
9. See page load correctly

**Expected behavior**
I expected the mdx file to load into my template page using latest gatsby-mdx

**Screenshots**
![image](https://user-images.githubusercontent.com/42518059/50569376-8eb07b80-0d31-11e9-8d1c-fc85fa23f587.png)

**Additional context**

**Comment by ChristopherBiscardi** [comment]
Yeah, it looks like you were using `MDXRenderer` without passing any `scope` in and without using the `componentWithMDXScope` APIs. By upgrading to `gatsby-mdx@0.3.1-ci.225` you raised the version you were using to the latest `ci` release, or effectively a recent master build. The more recent version removes the `componentWithMDXScope` API completely so that you don't need to worry about it any more, and in addition, that version of `MDXRenderer` imports `React` and `MDXTag` so that you don't have to think about it anymore.

Basically, by upgrading you've removed the problem which is really good news. If you keep using the latest `ci` tag, which is `0.3.1-ci.234` right now, you won't have this problem again.

If you decide to move back to `0.2.0` for some reason, you'll either have to use `componentWithMDXScope` when using `createPage` or `MDXRenderer scope={{React, MDXTag}}` for it to work. I suggest using the `ci` releases instead as they're pretty close to being the stable release anyway.

**Comment by ChristopherBiscardi** [comment]
@btables no worries! I'm in the process of redoing the docs and they will be released with the current master release soonish. #210

**Comment by deltaskelta** [comment]
How can I find what the latest ci build is? I can't seem to find them listed anywhere
